src/OSPSuite.Core/Comparison/TableFormulaDiffBuilder.cs

```diff
@@ -28,14 +23,8 @@ public override void Compare(IComparison<TableFormula> comparison)
          CompareStringValues(x => x.XName, x => x.XName, comparison);
          CompareStringValues(x => x.YName, x => x.YName, comparison);
          CompareValues(x => x.UseDerivedValues, x => x.UseDerivedValues, comparison);
-         _enumerableComparer.CompareEnumerables(comparison, x => x.AllPoints(), (pt1, pt2) => ValueComparer.AreValuesEqual(pt1.X, pt2.X, comparison.Settings.RelativeTolerance), x => displayValueFor(x, comparison));
-      }
-
-      private string displayValueFor(ValuePoint valuePoint, IComparison<TableFormula> comparison)
-      {
-         var formulaOwnerName = _displayNameProvider.DisplayNameFor(comparison.CommonAncestor);
-         var tableFormula = comparison.Object1.DowncastTo<TableFormula>();
-         return Captions.Comparisons.ValuePointAt(tableFormula.Name, formulaOwnerName, tableFormula.XDisplayValueFor(valuePoint.X));
+         //We do not use enumerable comparison here as we need to compare the values by index
+         _enumerableComparer.CompareEnumerablesByIndex(comparison, x => x.AllPoints);
```

Comment:
here we compare by index. That way comparison of tables with two points with very close x axis does not fail for no reason